build.sh

```diff
@@ -163,6 +163,53 @@ generate_rpm_sources() {
         .
 }
 
+get_tomcat_app_server() {
+    release_file="/etc/os-release"   
+    fedora_cutoff=$1
+    rhel_cutoff=$2
+    source "$release_file"
```

Comment:
I'd rather not use `source` since IIUC it will load the everything into the global namespace, potentially interfering with other things. I'd suggest to use `sed` to get just the `VERSION_ID` from the file.